service/ziti-tunnel/service/state.go

```diff
@@ -522,6 +544,7 @@ func (t *RuntimeState) RemoveZitiTun() {
 		if err != nil {
 			log.Errorf("Error deleting already existing interface: %v", err)
 		} else {
+			tun_release.Store(success)
```

Comment:
i probably should have made this function private not public. i also didn't expect to see the 'tun_release' used here. it should really all be in the .close function ?